src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -432,7 +441,7 @@ ucs_status_t uct_rc_verbs_ep_flush(uct_ep_h tl_ep, unsigned flags,
     }
 
     if (uct_rc_txqp_unsignaled(&ep->super.txqp) != 0) {
-        status = uct_rc_verbs_ep_post_flush(ep);
+        status = uct_rc_verbs_ep_post_flush_safe(ep, IBV_SEND_SIGNALED);
```

Comment:
can we remove uct_rc_verbs_ep_post_flush_safe and put its code in here?

Reply:
yes, merged into single function